src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -130,8 +130,8 @@ private static function isExpired(string $date): bool
     private static function getDotenvVars(): array
     {
         $vars = [];
-        foreach (explode(',', getenv('SYMFONY_DOTENV_VARS')) as $name) {
-            if ('' !== $name && false !== $value = getenv($name)) {
+        foreach (explode(',', $_ENV['SYMFONY_DOTENV_VARS']) as $name) {
+            if ('' !== $name && false !== $value = $_ENV[$name]) {
```

Comment:
`false !== $value = $_ENV[$name]`
You should check `isset($_ENV[$name])`, not is it false.

Reply:
Fixed